src/storage/src/flush.rs

```diff
@@ -293,7 +297,7 @@ impl<S: LogStore> FlushJob<S> {
             .flatten()
             .collect();
 
-        logging::info!("Successfully flush memtables to files: {:?}", metas);
+        logging::info!("Successfully flush memtables, region:{region_id}, files: {metas:?}");
```

Comment:
The file_metas is large, and the log is annoying.

Reply:
> The file_metas is large, and the log is annoying.

Maybe print only the file_id of metas?